embark-theme.conf

```diff
@@ -0,0 +1,53 @@
+# Embark (https://github.com/embark-theme/kitty) {{{
+background #1e1c31
+foreground #EEFFFF
+cursor #A1EFD3
+selection_background  #fbfcfc
+
+# black
+color0 #1e1c31
+color8 #3E3859
```

Comment:
This color has just recently changed. Can you update this to #585273?

Reply:
Sure thing.